src/rmkit/ui/events.cpy

```diff
@@ -92,7 +121,11 @@ namespace ui:
       if not long_press.empty():
         long_press_timer = ui::set_timeout([=]() {
           auto ev_copy = ev
-          finish(long_press, ev_copy)
+          is_long_press = true
+          dispatch(long_press, ev_copy)
+          // set up LONG_DOWN
+          prev_ev = ev_copy
+          state = LONG_DOWN
```

Comment:
I originally made this opt-in (i.e. you'd have to call `gestures.use_long_press_and_drag(true)` to set it up), but I think it's reasonable to have it always on and have the handler distinguish between regular and long drags w/ `ev.is_long_press`